docs/SPIR-V.rst

```diff
@@ -1865,6 +1865,15 @@ mode that is used in order to represent the given primitive type.
 |``triangleadj``      | ``InputTrianglesAdjacency`` |
 +---------------------+-----------------------------+
 
+Translation of Output Stream Types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Supported output stream types in geometry shaders are: ``PointStream<T>``,
+``LineStream<T>``, and ``TriangleStream<T>``. These types are translated to the underlying type ``T``.
```

Comment:
We also need to generate the correct execution mode from these types: OutputPoints, OutputLineStrip, or OutputTriangleStrip.